packages/engine/src/datastore/batch/metaversion.rs

```diff
@@ -33,15 +210,70 @@ impl Metaversion {
         self.batch
     }
 
+    /// Assert invariants, given that `version` is a metaversion of
+    /// *the same batch* as `self`.
+    // TODO: Actually this only applies to persisted metaversions, because loaded metaversions can
```

Comment:
Done

Reply:
Sorry I meant can we literally add the module and line number (or similar) to the task so we know where the TODO is. It would be too easy to miss some of these TODOs

Reply:
Added the TODO references to the task